wrapper/pre_copy.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, data):
 
         self.server = uri.hostname
         self.port = uri.port
-        self.user = 'administrator@vsphere.local'
+        self.user = 'root'
```

Comment:
I know I said I'm fine with it, but I figured it would make more sense (for users/developers, not management apps) to have this value based on the scheme, `esx` -> `root` and `vpx` -> `administrator@vsphere.local` would make sense.